stdlib/Test/test/runtests.jl

```diff
@@ -633,22 +633,36 @@ end
 end
 
 @testset "file info in test errors" begin
-     local f = tempname()
-
-     write(f,
-     """
-     using Test
-     @testset begin
-         @test 1==2
-         @test_throws UndefVarError 1
-         @test_broken 1 == 1
-     end
-     """)
-
-     local msg = read(pipeline(ignorestatus(`$(Base.julia_cmd()) --startup-file=no --color=no $f`), stderr=DevNull), String)
-     @test contains(msg, "at " * f * ":" * "3")
-     @test contains(msg, "at " * f * ":" * "4")
-     @test contains(msg, "at " * f * ":" * "5")
-
-     rm(f; force=true)
+    local f = tempname()
+
+    write(f,
+    """
+    using Test
+    @testset begin
+        @test 1==2
+        @test_throws UndefVarError 1
+        @test_broken 1 == 1
+    end
+    """)
+
+    local msg = read(pipeline(ignorestatus(`$(Base.julia_cmd()) --startup-file=no --color=no $f`), stderr=DevNull), String)
+    @test contains(msg, "at " * f * ":" * "3")
+    @test contains(msg, "at " * f * ":" * "4")
+    @test contains(msg, "at " * f * ":" * "5")
+
+    rm(f; force=true)
  end
+
+# issue #24919
+@testset "≈ with atol" begin
+    src = """
+    using Test
+    x, y = 0.9, 0.1
+    @test x ≈ y atol=0.01
+    """
+    msg = read(pipeline(
+        ignorestatus(`$(Base.julia_cmd()) --startup-file=no --color=no -e $src`),
+        stderr=DevNull), String)
+
+    @test contains(msg, "Evaluated: 0.9 ≈ 0.1; atol=0.01")
```

Comment:
Maybe put the `atol=0.01` in parens or something like that?

Reply:
you want something like `0.9 ≈ 0.1 (atol=0.01, ...)`?

Reply:
Yes, something like that – the semicolon just looks weird there for some reason.

Reply:
ok, done